FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt

```diff
@@ -944,16 +1073,17 @@ object PgpMsg {
         )
       }
     } else {
-      if (PgpDecrypt.DecryptionErrorType.NO_MDC == decryptionResult.exception.decryptionErrorType) {
-        val resultWithIgnoredMDCErrors = PgpDecrypt.decryptWithResult(
-          msgBlock.content?.toInputStream()!!,
-          ringCollection,
-          protector,
-          true
+      if (PgpDecryptAndOrVerify.DecryptionErrorType.NO_MDC == decryptionResult.exception.decryptionErrorType) {
+        val resultWithIgnoredMDCErrors = PgpDecryptAndOrVerify.decryptAndOrVerifyWithResult(
+          srcInputStream = msgBlock.content?.toInputStream()!!,
+          pgpPublicKeyRingCollection = pgpPublicKeyRingCollection,
+          pgpSecretKeyRingCollection = pgpSecretKeyRingCollection,
+          protector = protector,
+          ignoreMdcErrors = true
         )
```

Comment:
Ignoring MDC errors should only happen after user confirmed that they want to take the risk. Or do you pre-decrypt it while ignoring MDC but don't render first?

Should add a comment as this is security sensitive.

Reply:
@tomholub 
Please look at these screenshots

![image](https://user-images.githubusercontent.com/2863246/146149419-9e8c14f4-81b5-4496-a5ad-56d4d71ab963.png)

> Or do you pre-decrypt it while ignoring MDC but don't render first?

It was simpler to do

> Ignoring MDC errors should only happen after user confirmed that they want to take the risk. 

Should I change the logic to use this way?

Reply:
@tomholub What about this one? It seems it's the last one for the current PR.